tools/message-screenshot.js

```diff
@@ -0,0 +1,66 @@
+const puppeteer = require("puppeteer");
+const commander = require("commander");
+const path = require("path");
+const host = "localhost:9991";
+const options = {};
+
+commander
+    .arguments('<integration> <message_id>')
+    .action((integration, messageId) => {
+        options.integration = integration;
+        options.messageId = messageId;
+        console.log(`Capturing screenshot for ${integration} using message ${messageId}`);
+    })
+    .parse(process.argv);
+
+if (options.integration === undefined) {
+    console.error('no integration specified!');
+    process.exit(1);
+}
+
+async function run() {
+    const browser = await puppeteer.launch({
+        args: [
+            '--window-size=1400,1024',
+            '--no-sandbox', '--disable-setuid-sandbox',
+            '--font-render-hinting=none',
+        ],
+        defaultViewport: null,
+        headless: true,
+    });
+    try {
+        const page = await browser.newPage();
+        await page.setViewport({ width: 1280, height: 1024, deviceScaleFactor: 2 });
+        await page.goto('http://' + host);
+        // wait for devlogin admin button and click on it
+        await page.waitForSelector('.btn-admin');
+        await page.click('.btn-admin');
+
+        // Navigate to message and capture screenshot
+        await page.goto(`http://${host}/#narrow/near/${options.messageId}`);
+        const messageSelector = `#zfilt${options.messageId}`;
+        await page.waitForSelector(messageSelector);
+        // remove unread marker and don't select message
+        const marker = `#zfilt${options.messageId} .unread_marker`;
+        await page.evaluate((sel) => $(sel).remove(), marker);  // eslint-disable-line no-undef
+        await page.evaluate(() => navigate.up());  // eslint-disable-line no-undef
+        const messageBox = await page.$(messageSelector);
+        const messageGroup = (await messageBox.$x('..'))[0];
+        // Compute screenshot area, with some padding around the message group
+        const clip = Object.assign({}, await messageGroup.boundingBox());
+        clip.y -= 5;
+        clip.x -= 5;
+        clip.width += 10;
+        clip.height += 10;
+        const imagePath = path.join(__dirname, '..', 'static', 'images', 'integrations', options.integration, '001.png');
+        await page.screenshot({ path: imagePath, clip: clip });
+        console.log(`Captured screenshot for ${options.integration}!`);
```

Comment:
This should print the path, maybe with some color, at the end so it's easy to find :)